twitter.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+        <link rel="stylesheet" href="twitter.css" />
+        <title>Image Grid</title>
+    </head>
+
+    <body>
+    <div class="main-grid">
+        <div class="left-nav">
+            <div class="logo">
+                <img class="logo" src="./images/twitter-154-svgrepo-com.svg" alt="twitter" />
+            </div>
+            <div class="nav">
+                <div class="item">
+                    <img class="logo" src="./images/home.svg"><a href="#">Home</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/explore.svg"><a href="#">Explore</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/notifications.svg"><a href="#">Notifications</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/messages.svg"><a href="#">Messages</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/bookmark.svg"><a href="#">Bookmarks</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/lists.svg"><a href="#">Lists</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/profile.svg"><a href="#">Profile</a>
+                </div>
+                <div class="item">
+                    <img class="logo" src="./images/more.svg"><a href="#">More</a>
+                </div>
```

Comment:
Here is some wrong because you are using tag a only for text and if you try to click on it it will be work only when you will click on the text and if you click on the icon which you put here it will not work. So try to create a tag and inside this tag put img and span tag for text.

```html
<a href="#">
    <img class="logo" src="./images/bookmark.svg">
    <span>Bookmarks<span>
</a>
```